### DISCUSSION TITLE
Menu 过长时滚动而不是压缩内容

### DISCUSSION THREAD

**Discussion by qiaoshouzi** [tusen-ai/naive-ui#3190] [Q&A]
Menu 过长时会压缩内容，有没有办法可以像 Tabs 一样可以滚动

![gif](https://user-images.githubusercontent.com/61568944/175843117-bbe31295-2e9d-4186-9ad7-51b43d34097c.gif)

**Comment by 07akioni** [comment]
未来估计不会这么处理，一直想做折叠但是有点复杂，没空搞
